spec.html

```diff
@@ -12752,10 +12752,7 @@ <h1>Static Semantics: Early Errors</h1>
       </emu-grammar>
       <ul>
         <li>
-          It is an early Reference Error if AssignmentTargetType of |LeftHandSideExpression| is ~invalid~.
-        </li>
-        <li>
-          It is an early Syntax Error if AssignmentTargetType of |LeftHandSideExpression| is ~strict~.
+          It is an early Syntax Error if AssignmentTargetType of |LeftHandSideExpression| is ~invalid~ or ~strict~.
```

Comment:
Note: it seems that there's also precedent for writing this as `is not ~simple~`, if we'd prefer.